COVIDSafepassage/passsystem/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.0.4 on 2020-04-11 06:16
+# Generated by Django 3.0.4 on 2020-04-20 12:05

```

Comment:
this file shouldn't be affected. new models should result in new migration files. Old files should be kept intact. Else we'll have to drop database and restart.

Reply:
Actually, in local machine, first I tried to migrate as next migration file. But it gave some errors, like unable to migrate. So, I had to delete all migration files and put the latest as initial migration file.

Reply:
That's not the right way. On a fresh local database, you should be able to `manage.py migrate`. And then with your changes, `manage.py makemigrations`. Else we'll have to throw away the server on google cloud and restart.